src/commands/archive-card.js

```diff
@@ -0,0 +1,178 @@
+"use strict";
+
+var List = require('term-list');
+  var prompt = require('prompt');
+
+var __ = function(program, output, logger, config, trello, translator) {
+
+    var trelloApiCommand = {};
+
+    trelloApiCommand.makeTrelloApiCall = function(options, onComplete) {
+        if (!options.match) {
+            options.match = 'exact';
+            logger.warning('--match parameter not specified, defaulting to exact title match');
```

Comment:
A user should've read the "usage" and know that exact is the default, so I'm actually thinking to remove this completely